_episodes_rmd/08-plot-ggplot2.Rmd

```diff
@@ -235,15 +235,15 @@ The `scale_x_log10` function applied a transformation to the coordinate system o
 We can fit a simple relationship to the data by adding another layer,
 `geom_smooth`:
 
-```{r lm-fit}
+```{r lm-fit, fig. alt = "Scatter plot of life expectancy vs GDP per capita with a blue trend line summarising the relationship between variables, and gray shaded area indicating 95% confidence intervals for that trend line."}
 ggplot(data = gapminder, mapping = aes(x = gdpPercap, y = lifeExp)) +
   geom_point(alpha = 0.5) + scale_x_log10() + geom_smooth(method="lm")
 ```
 
 We can make the line thicker by *setting* the **size** aesthetic in the
 `geom_smooth` layer:
 
-```{r lm-fit2}
+```{r lm-fit2, fig. alt = "Scatter plot of life expectancy vs GDP per capita with a trend line summarising the relationship between variables. The blue trend line is slightly thicker than in the previous figure."}
```

Comment:
```suggestion
```{r lm-fit2, fig.alt = "Scatter plot of life expectancy vs GDP per capita with a trend line summarising the relationship between variables. The blue trend line is slightly thicker than in the previous figure."}
```